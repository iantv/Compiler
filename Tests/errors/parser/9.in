struct point{
	int x;
	int y;
}b;
int main(){
	return b;
}