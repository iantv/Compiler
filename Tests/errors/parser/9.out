error C2440: cannot convert "struct point" to "int" in line 6 column 10
